tests/python/relay/test_pass_mac_count.py

```diff
@@ -0,0 +1,101 @@
+"""Unit tests for MAC counter."""
+import tvm
+from tvm import relay
+import sys
+sys.path.append('../frontend/mxnet')
```

Comment:
Sorry, why do we need to add it to the system path?

Reply:
This is to import the model_zoo from this path correctly. I am happy to change to a more elegant solution if any suggestion.

Reply:
I actually suggest to remove the resnet test, as no one can verify the correctness of the test by eyeball. And later if the algorithm changes or enhanced, it will also be a pain to deal with such test.